kong/plugins/prometheus/exporter.lua

```diff
@@ -157,6 +158,11 @@ local function init()
                                       "AI requests cost per ai_provider/cache in Kong",
                                       {"ai_provider", "ai_model", "cache_status", "vector_db", "embeddings_provider", "embeddings_model", "token_type", "workspace"})
 
+  metrics.ai_llm_latency = prometheus:histogram("ai_llm_latency_ms",
+                                      "Latency added by llm response ai_provider in Kong",
```

Comment:
this would not be accurate description, as post_request would be called multiple times, the last one is after body_filter. so this is actually the whole request latency. I'm wondering is it what we are looking for?

Reply:
@fffonion  this line is inside `for _, ai_plugin in pairs(serialized.ai_metrics) do`
So it will be the llm response latency taken by an ai provider and model

Reply:
Please let me know if the new description is ok. Thanks

Reply:
Yes, I do agree it's response latency, but I'm wondering whether it's the latency for the whole response or the latency added by kong operations only